spec/user_interface_spec.rb

```diff
@@ -6,7 +6,7 @@
   let(:output) { StringIO.new }
   let(:valid_input) { StringIO.new('1') }
 
-  it 'prints menu of options for user to choose' do
+  it 'Prints menu of options for user to choose' do
```

Comment:
test descriptions start with lowercase
that is because of how the test reads when you run it in the terminal:

`User interface prints menu of options for user to choose`